packages/sign-client/src/constants/engine.ts

```diff
@@ -102,3 +102,8 @@ export const ENGINE_RPC_OPTS: EngineTypes.RpcOptsMap = {
     },
   },
 };
+
+export const SESSION_REQUEST_EXPIRY_BOUNDARIES = {
+  min: 300,
```

Comment:
Can we use the labelled values from `@walletconnect/time` here instead of inline numbers as above? Should have both 5 mins and 7 days available already as far as I remember